CHANGELOG.md

```diff
@@ -11,7 +12,10 @@
 ...
 
 ### Breaking Changes
-...
+- The way you configure indexes in the `DocumentIndexClient` has changed:
+    - The `EmbeddingType` alias has been renamed to `Representation` to better align with the underlying API.
+    - The `embedding_type` field has been removed from the `IndexConfiguration` class. You now configure embedding-related parameters via the `embedding` field.
+    - You now always need to specify an embedding model when creating an index. Previously, this was always `luminous-base`.
```

Comment:
I think we should add a line like "For examples of the new usage, see the document_index.ipynb" with a link to the notebook to make the migration easier

Reply:
Thanks for the suggestion, will update this :)